lib/ansible/modules/system/gather_facts.py

```diff
@@ -25,8 +25,8 @@
 
 DOCUMENTATION = '''
 ---
-module: setup
-version_added: historical
+module: gather_facts
+version_added: 2.4
```

Comment:
It's too bad we don't have a way (aside from the description) to indicate a different version_added for the new and old names.